packages/hash/api/src/collab/graphql/queries/blocks.queries.ts

```diff
@@ -33,3 +33,21 @@ export const getBlocksQuery = gql`
 
   ${blockFieldsFragment}
 `;
+
+export const knowledgeBlockFieldsFragment = gql`
+  fragment KnowledgeBlockFields on KnowledgeBlock {
+    __typename
+    entityId
+    entityVersion
+    accountId
```

Comment:
Can we either query `ownedById` instead of `accountId`, or in addition?

Reply:
I would be a little bit cautious to do so here, as it would likely bubble into all the collab block types